### DISCUSSION TITLE
Changing state of parent using flutter hooks

### DISCUSSION THREAD

**Discussion by thunderstorm010** [rrousselGit/flutter_hooks#271] [Q&A]
When building an theme changing app, for example, we would make a method in the parent widget like;
```dart
static _MyAppState of(BuildContext context) => 
      context.findAncestorStateOfType<_MyAppState>();
```
and write a helper method for changing the state.
How can this be achieved in flutter-hooks?

**Comment by rrousselGit** [comment]
You should use an Inheritedwidget, such as Provider
